x-pack/legacy/plugins/alerting/common/types.ts

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { SavedObjectAttributes } from 'kibana/server';
+import { AlertActionParams } from '../server/types';
+
+export interface IntervalSchedule extends SavedObjectAttributes {
+  interval: string;
```

Comment:
Can we change interval type from the string to duration? We have a lot of complexity on the schema check (RegExp) per each API request - `interval: getDurationSchema().required(),`and in the new platform it will be simpler with `interval: schema.duration()`. Not sure how it impact the UI here

Reply:
I've actually said the same in the past - we should use the schema validation on everything returned by SavedObjects Client - but I think that should be a separate PR.